workflow/rocoto/tasks.py

```diff
@@ -149,6 +149,11 @@ def _get_forecast_hours(run, config, component='atmos') -> List[str]:
             local_config['FHOUT_GFS'] = config['FHOUT_ICE_GFS']
             local_config['FHOUT'] = config['FHOUT_ICE']
 
+        if component in ['wave']:
+            local_config['FHOUT_HF_GFS'] = config['FHOUT_HF_WAV']
+            local_config['FHOUT_GFS'] = config['FHOUT_HF_WAV']
```

Comment:
This is not correct
```suggestion
            local_config['FHOUT_GFS'] = config['FHOUT_WAV']
```

I'm not sure what the output frequency should be for gdas (`FH_OUT`). We don't have a separate setting for gdas wave output, just `FHOUT_WAV`.